packages/vm/src/checksum.rs

```diff
@@ -46,14 +43,38 @@ impl TryFrom<&[u8]> for Checksum {
     }
 }
 
+impl Into<Vec<u8>> for Checksum {
+    fn into(self) -> Vec<u8> {
+        self.0.into()
```

Comment:
This is potentially faster than the old `to_vec()` implementation, since it consumes owned `self`. But that's up to the compiler of course.

Reply:
oh, this is only available in Rust 1.43+. Will update